spyder/plugins/projects/plugin.py

```diff
@@ -213,18 +215,19 @@ def register(self):
                 lambda v: outline_explorer.update_all_editors())
         if self.main:
             self.sig_project_closed.connect(
-                lambda v: self.main.set_window_title())
+                lambda v, s: self.main.set_window_title())

```

Comment:
Please revert this change. The thing is when not setting the signature in a signal, then it uses the first one as default (i.e. the same as before).